corehq/util/test_utils.py

```diff
@@ -232,6 +234,23 @@ def inner(*args, **kwargs):
     return inner
 
 
+class OverridableSettingsTestMixin(object):
+    """Backport of core Django functionality to 1.7. Can be removed
+    once Django >= 1.8
+    https://github.com/django/django/commit/d89f56dc4d03f6bf6602536b8b62602ec0d46d2f"""
+    @classmethod
+    def setUpClass(cls):
+        if cls._overridden_settings:
```

Comment:
What do you think of adding a usage example in the doc string. Something like:
```
Usage:

  @override_settings(A_SETTING=True)
  class SomeTests(TestCase, OverridableSettingsTestMixin):

      @classmethod
      def setUpClass(cls):
          super(SomeTests, cls).setUpClass()
          # settings.A_SETTING is True here

      @classmethod
      def tearDownClass(cls):
          # teardown stuff
          # don't forget to call super to undo override_settings
          super(SomeTests, cls).tearDownClass()
```
It's not immediately obvious that this is connected to the `override_settings` class decorator and that you need to call `super` when overriding `setUpClass`/`tearDownClass`.